src/nlp/commands/run_beam.py

```diff
@@ -7,8 +7,8 @@
 
 from nlp.builder import FORCE_REDOWNLOAD, HF_DATASETS_CACHE, REUSE_CACHE_IF_EXISTS, DatasetBuilder, DownloadConfig
 from nlp.commands import BaseTransformersCLICommand
+from nlp.info import DATASET_INFOS_DICT_FILE_NAME
 from nlp.load import import_main_class, prepare_module
-from nlp.utils.info_utils import DATASET_INFOS_DICT_FILE_NAME
```

Comment:
@lhoestq this threw an error for me when running a command

Reply:
good catch !